app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -340,14 +340,7 @@ class BrowserFragment : EngineViewLifecycleFragment(), Session.Observer {
 
         if (event.keyCode == KeyEvent.KEYCODE_DPAD_CENTER && actionIsDown) MenuInteractionMonitor.selectPressed()
 
-        /**
-         * [Session.url] is currently set late in the loading process, so while loading the first
-         * site of a session [isUrlEqualToHomepage] will return true for some time (often several
-         * seconds).  We check [Session.loading] to prevent the menu button from being unresponsive
-         * during that time
-         */
-        val overlayCanBeInvisible = !isUrlEqualToHomepage || session.loading
-        if (event.keyCode == KeyEvent.KEYCODE_MENU && overlayCanBeInvisible) {
+        if (event.keyCode == KeyEvent.KEYCODE_MENU && !isUrlEqualToHomepage) {
```

Comment:
Session.url no longer loads slowly, so this hack can be cleaned out.

Reply:
For future reference, this type of comment is best put into a small, standalone commit with the comment in the commit summary so that this information is captured in version control.

Reply:
This also would have been a great commit to backout. :)